source/test/TestApp/TestReport.cpp

```diff
@@ -81,6 +81,12 @@ namespace PlayFabUnit
             case TestFinishState::PASSED: internalReport.passed += 1; break;
             case TestFinishState::FAILED: internalReport.failures += 1; break;
             case TestFinishState::SKIPPED: internalReport.skipped += 1; break;
+            case TestFinishState::PENDING:
```

Comment:
Why are we handling these two states with 'break' instead of letting default handle them.  If it's for the sake of handling all possible states, default should probably log or assert as it would be handling a completely invalid value.

Reply:
the warning on iOS was that there were defined enums that weren't being caught. I was considering letting each fall through to default, but recalled one compiler threw a warning when it came across the fall through. This way we don't see any warnings, and the compiler should be optimizing these empty statements away.